docs/bucket/notifications/README.md

```diff
@@ -431,6 +431,7 @@ An example of Redis configuration is as follows:
 "redis": {
     "1": {
         "enable": true,
+	"format": "namespace",
```

Comment:
Please fix the indentation here.

Reply:
^^ emacs does this even with markdown mode enabled. Will check